apps/web/src/pages/settings/WebhookPage/WebhookPage.tsx

```diff
@@ -0,0 +1,152 @@
+import { Button, Input, inputStyles } from '@novu/design-system';
+import { FC } from 'react';
+import { Controller } from 'react-hook-form';
+
+import { ClipboardIconButton } from '../../../components';
+import { Timeline } from '../../../components/Timeline';
+import { css, cx } from '../../../styled-system/css';
+import { HStack, Stack } from '../../../styled-system/jsx';
+import { text } from '../../../styled-system/recipes';
+import { SettingsPageContainer } from '../SettingsPageContainer';
+import { Text } from './WebhookPage.shared';
+
+// Unfortunately, a wrapper around TimelineItem prevented any styles from applying, so have to use direct import for now
+import { Timeline as MantineTimeline } from '@mantine/core';
+import { useWebhookPage } from './useWebhookPage';
+import { WebhookClaimStatusDisplay } from './WebhookClaimStatusDisplay';
+import { WebhookAdditionalInformationLink } from './WebhookAdditionalInformationLink';
+
+const codeValueInputClassName = css({
+  '& input': {
+    border: 'none !important',
+    background: 'surface.popover !important',
+    color: 'typography.text.secondary !important',
+    fontFamily: 'mono !important',
+  },
+});
+
+// source: https://www.geeksforgeeks.org/how-to-validate-a-domain-name-using-regular-expression/
+const DOMAIN_REGEX = /^(?!-)[A-Za-z0-9-]+([\-\.]{1}[a-z0-9]+)*\.[A-Za-z]{2,6}$/;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Should we have this restriction? Currently, we have 0 server-side restriction on domains 😱‏

Reply:
I also noticed the regex doesn't support URL schemes (http, https etc)

Reply:
Correct -- from my understanding (and how its labeled on the page), this should be a domain name, not an URL